SPECS/libtpms/libtpms.spec

```diff
@@ -0,0 +1,347 @@
+Name:           libtpms
+Version:        0.9.6
+Release:        4%{?dist}
```

Comment:
should version=1?  this is the first version for mariner ... 4 comes from photon.

Reply:
maybe it should be 5?

Reply:
5 is the preference.